app/controllers/users_controller.rb

```diff
@@ -16,9 +16,8 @@ def show
   end
 
   def new
-    if logged_in?
-      redirect_to root_url
-    end
+    return redirect_to root_path if logged_in?
+
     @user = User.new
```

Comment:
Looks OK, but line 21 should move up before the return

Reply:
The reason the return is before setting the @user variable is that when we redirect there is no need to set that variable at all.